### DISCUSSION TITLE
Exif information is missing after upload

### DISCUSSION THREAD

**Discussion by soana** [photoprism/photoprism#2793] [General]
I tried to upload pictures from my Android phone (S21) by using the Upload option from the PhotoPrism app but the geotag information is missing on the picture transferred to Photoprism.
Same thing happened when using the third party android app: "Photo Uploader for PhotoPrism" 

Both ways the geotag information is being stripped - not the same happens when I transfer the picture on a PC (Win10) and use the Upload function or the Import function from any browser - in my case I used Firefox and Safari.

**Comment by lastzero** [comment]
Likely a phone setting / feature, although we do not have experience with your particular device.

**Reply by soana** [reply]
Thanks for the quick feedback - I agree that this looks more like a phone issue since it's only happens when transferring directly from the phone to my locally hosted PhotoPrism container. 
Will run some more DOE's and report back here.

**Reply by mendoncart** [reply]
Hi @soana!

Did you find a way to upload the photos with location metadata?
I'm having the same problem: when I upload photos using photoprism app, or through a third party app, like emby, they all come without location. 

I agree that is more likely a phone setting or something that is blocking the sharing of the location for security reasons, but I can't find how to enable location sharing for my photoprism server...

The only way I found to transfer pictures with the location, is to plug my phone directly to the computer and copy the files.
But I'd like to have a more convenient solution.

**Reply by soana** [reply]
I started using nextcloud to transfer pictures from phone to my server in a location that photoprism is checking every hour for new pictures. The process is not instant but it's automated. 

As of now I have not found any other way besides the USB transfer to keep the location information with the phone pictures.

**Reply by lastzero** [reply]
You can use any webdav compatible app or operating system to upload files.

**Reply by mendoncart** [reply]
I don't know if the developers of the photoprism android app are the same of the photoprism, but the problem is within file permissions on android. The app must request the correct permission to access media and its metadata.

I'm using syncthing and it's working great to transfer the files with metadata.

**Reply by lastzero** [reply]
Nope, it's a community / third party project as all other native apps.
